### PR TITLE
Added documentation for brain-tumor-segmentation-0001

### PR DESCRIPTION
**Pull request by lachinov**

### CODE REVIEW

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| MParams           | 38.192        |
+| Source framework  | MXNet         |
+
+## Accuracy
+
+See [https://github.com/lachinov/brats2018-graphlabunn]()
+
+## Performance
+
+## Input
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: Add subsections `Original model`, `Converted model` with description of input/output of original and converted model separately, even if they are same. See [PR 174](https://github.com/opencv/open_model_zoo/pull/174) for example.

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+The channels are ordered as `T1`, `T2`, `T1ce`, `Flair`.
+
+## Output
+
+Tensor of shape `1,4,128,128,128` with probabilities of the given voxel to be in the `background`, `necrotic core`, `edema` or `enhancing tumor` class respectively.
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: Use pattern `<short description>, name - <output layer name>, shape - <shape>, <full description>` as in [174](https://github.com/opencv/open_model_zoo/pull/174)
Also add layer name to input description

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Brain-tumor-segmentation-0001
+
+## Use Case and High-Level Description
+This model was created for participation in the [Brain Tumor Segmentation Challenge](https://www.med.upenn.edu/sbia/brats2018.html) (BraTS) 2018.
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: Add blank line after heading

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Accuracy
+
+See [https://github.com/lachinov/brats2018-graphlabunn]()
+
+## Performance
+
+## Input
+
+The input is cropped, resamped and z-score normalzied image.
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: Please, provide more info about input preprocessing

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## Performance
+
+## Input
+
+The input is cropped, resamped and z-score normalzied image.  
+Image, shape - `1,4,128,128,128`, format is `B,C,D,H,W` where:
+
+- `B` - batch size
+- `C` - channel
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: Is it really channel, or it has more suitable name?

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md (hunk 2 above)
Comment (eizamaliev) [reply models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: <short description> = 'Tensor', for example

File: models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md (hunk 5 above)
Comment (lachinov) [reply models/public/brain-tumor-segmentation-0001/brain-tumor-segmentation-0001.md]: We can call it a modality, however a channel is also fine.

**Comment by eizamaliev** [comment]
Looks good

**Comment by eizamaliev** [comment]
@alinaalborova please review

**Comment by kchechil** [comment]
Jenkins please retry a build

**Comment by kchechil** [comment]
Jenkins please retry a build

**Pull request merged**